fe2-android/app/src/main/java/com/github/dedis/popstellar/repository/remote/LAORequestFactory.java

```diff
@@ -0,0 +1,31 @@
+package com.github.dedis.popstellar.repository.remote;
+
+import androidx.annotation.NonNull;
+
+import com.tinder.scarlet.websocket.okhttp.request.RequestFactory;
+
+import okhttp3.Request;
+
+public class LAORequestFactory implements RequestFactory {
+
+  @NonNull private static String url;
+
+  public LAORequestFactory(@NonNull String defaultUrl) {
+    url = defaultUrl;
+  }
```

Comment:
Each new instance of LAORequestFactory will modify the url for all instances.
If we only store the url as a static field, it would be better to modify url only with the static setUrl and not the constructor.
If only a single LAORequestFactory instance should exist, using the singleton pattern would be better perhaps.